### PR TITLE
Add ItemAction component

### PR DESCRIPTION
**Pull request by pith**
image

### CODE REVIEW

File: src/itemAction/index.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  readonly action?: string
+  readonly subLabel?: string
+  readonly leftAddon?: React.ReactNode
+  readonly onClick?: (event: React.MouseEvent<HTMLElement>) => void
+  readonly onBlur?: (event: React.FocusEventHandler<HTMLElement>) => void
+  readonly onFocus?: (event: React.FocusEventHandler<HTMLElement>) => void
+  readonly onMouseDown?: (event: React.MouseEvent<HTMLElement>) => void
+}
+
+class ItemAction extends PureComponent<ItempActionProps> {
```
Comment (Bidjii) [inline src/itemAction/index.tsx]: Shouldn't this be a functional component?

Comment (Bidjii) [reply src/itemAction/index.tsx]: And why is it `ItempActionProps` and not `ItemActionProps`? ;)

File: src/_utils/item/style.ts [hunk 2]
```diff
@@ -1,19 +1,40 @@
...
+import { color, space } from '_utils/branding'
 
 export default css`
   :global(.kirk-item) {
     position: relative;
     display: flex;
     padding: 0;
-    padding-top: ${space.l};
-    padding-bottom: ${space.l};
+    padding: ${space.l} ${space.xl};
```
Comment (Bidjii) [inline src/_utils/item/style.ts]: This is huge, it's going to impact all the Items throughout the lib (not only the `ItemAction`). Not a problem per se, just wanted to point that out.

Comment (pith) [reply src/_utils/item/style.ts]: Yep I know that is why I listed it in the breaking changes

File: src/itemAction/index.tsx (hunk 1 above)
Comment (pith) [reply src/itemAction/index.tsx]: Fixed thanks for the feedback ! :)

File: src/_utils/item/style.ts [hunk 3]
```diff
@@ -1,19 +1,40 @@
...
     align-items: center;
     flex: 1;
     border: 0;
     background: none;
   }
+  :global(.kirk-item.kirk-item--clickable:hover) {
+    background: ${color.pressed};
```
Comment (Tuxosaurus) [inline src/_utils/item/style.ts]: why 'pressed' and not 'hover' (since it's not pressed - also if it was pressed, the status would be "active")

**Pull request merged**